repos/tap/components/list/treeList/treeList.js

```diff
@@ -81,24 +81,34 @@ const NodeComponent = ({ node, level, isExpanded, hasChildrenNodes }) => {
   // don't display empty folders
   if (level === 0 && isEmptyFolderNode(node)) return null
 
+  const { activeFile } = useStoreItems([CATEGORIES.ACTIVE_FILE])
   const theme = useTheme()
   const themeStyles = theme.get('treeList')
   const [ styleRef, mainStyles ] = useThemeHover(themeStyles?.default, themeStyles?.hover)
+  // check if active file or expanded folder
```

Comment:
Wrap all this into a use memo hook, that returns and object with property's main and text which are arrays with the correct styles.

Then just apply the arrays directly to the components